spec/services/subscriptions/override_service_spec.rb

```diff
@@ -120,4 +121,119 @@
       end
     end
   end
+
+  describe '.call_from_api' do
+    let(:external_customer_id) { customer.external_id }
+
+    let(:plan_params) do
+      {
+        name: plan_name,
+        code: "#{plan.code}-123456",
+        interval: plan.interval,
+        overridden_plan_id: plan.id,
+        pay_in_advance: plan.pay_in_advance,
+        organization_id: organization.id,
+        amount_cents: 299,
+        amount_currency: plan.amount_currency,
+        charges: [
+          {
+            billable_metric_id: billable_metric.id,
+            charge_model: 'standard',
+            properties: {
+              amount: '100',
+            },
+          },
+        ],
+      }
+    end
+
+    let(:subscription_params) do
+      {
+        external_customer_id: external_customer_id,
+        name: 'subscription display name',
+        external_id: SecureRandom.uuid,
+        billing_time: billing_time,
+      }
+    end
+
+    before { plan }
+
+    it 'creates a subscription' do
+      result = override_service.call_from_api(
+        organization: organization,
+        plan_args: plan_params,
+        subscription_args: subscription_params
```

Comment:
Style/TrailingCommaInArguments: Put a comma after the last parameter of a multiline method call. (https://rubystyle.guide#no-trailing-params-comma)